SqlKata.Execution/Query.Extensions.cs

```diff
@@ -168,6 +168,11 @@ public static int Insert(this Query query, IEnumerable<string> columns, IEnumera
             return CreateQueryFactory(query).Execute(query.AsInsert(columns, valuesCollection), transaction, timeout);
         }
 
+        public static async Task<int> InsertAsync(this Query query, IEnumerable<string> columns, IEnumerable<IEnumerable<object>> valuesCollection, IDbTransaction transaction = null, int? timeout = null)
```

Comment:
Do you mind add the `cancelationToken`?, (added in https://github.com/sqlkata/querybuilder/pull/487)
So I can approve this PR

Reply:
I will do it from my side